pom.xml

```diff
@@ -159,6 +163,114 @@
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>properties-maven-plugin</artifactId>
+                <version>1.0.0</version>
+                <executions>
+                    <execution>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>read-project-properties</goal>
+                        </goals>
+                        <configuration>
+                            <files>
+                                <file>${app.properties.path}</file>
+                            </files>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>io.fabric8</groupId>
+                <artifactId>docker-maven-plugin</artifactId>
+                <version>${docker-maven-plugin.version}</version>
+
+                <configuration>
+                    <dockerHost>unix:///var/run/docker.sock</dockerHost>
+
+                    <verbose>true</verbose>
+                    <images>
+                        <image>
+                            <name>boxitory</name>
+                            <build>
+                                <dockerFileDir>${project.basedir}/target/dockerfile/</dockerFileDir>
+
+                                <!--copies artficact to docker build dir in target-->
+                                <assembly>
+                                    <mode>dir</mode>
+                                    <descriptor>${project.basedir}/src/main/resources/assembly/release.xml</descriptor>
+                                </assembly>
+                                <tags>
+                                    <tag>latest</tag>
+                                    <tag>${project.version}</tag>
+                                </tags>
+                            </build>
+                            <run>
+                                <ports>
+                                    <!--suppress UnresolvedMavenProperty -->
+                                    <!--application.properties variable-->
+                                    <port>${server.port}:${server.port}</port>
+                                </ports>
+                                <volumes>
+                                    <bind>
+                                        <!--suppress UnresolvedMavenProperty -->
+                                        <volume>${app.boxes.home}:${box.container.home}</volume>
+                                        <!--suppress UnresolvedMavenProperty -->
+                                        <volume>${app.properties.path}:${box.container.home}/application.properties</volume>
+                                        <!--suppress UnresolvedMavenProperty -->
+                                        <volume>${app.logs}:${box.container.home}/logs</volume>
```

Comment:
Build is not failing at my local neither. Travis complains about logs dir. Shouldn't this line be without `/logs`? Same as on line `218` where also isn't `/boxes` directory.